components/shared/msg/constellation_msg.rs

```diff
@@ -466,7 +466,7 @@ pub enum InputMethodType {
 }
 
 #[derive(Clone, Copy, Debug, Deserialize, Serialize)]
-/// The equivalent of script_layout_interface::message::Msg
+/// The equivalent of script_layout_interface::Msg
```

Comment:
`Msg` is removed in this patch, so wouldn't this comment be incorrect? I guess this enum is now unused and can be deleted along with the `HangAnnotation::Layout` variant.  I'm fine if you think that can happenin a separate patch.

Reply:
You're right. I wasn't sure if these would be used again someday, but I think we should remove them now because they are dead code.

Thanks so much for the review!